vault/logical_system.go

```diff
@@ -1869,6 +1899,44 @@ func (b *SystemBackend) handleAuthTable(ctx context.Context, req *logical.Reques
 	return resp, nil
 }
 
+func augmentEnableAuthConfigMap(configMap map[string]interface{}) error {
```

Comment:
It feels like you could find a less generic name for what this does, e.g. "expandStringValsWithCommas" or something.

Reply:
Done